python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1203,13 +1202,10 @@ def clone_field(table, name, datatype):
 
 
 def test_multifile_warning(datadir):
-    fpath = datadir.__fspath__() + "/*.parquet"
-    with pytest.warns(UserWarning):
-        got = cudf.read_parquet(fpath)
-        fname = sorted(glob(fpath))[0]
-        expect = pd.read_parquet(fname)
-        expect = expect.apply(pd.to_numeric)
-        assert_eq(expect, got)
+    fpath = datadir.__fspath__() + "/multipart_parquet"
+    got = cudf.read_parquet(fpath + "/*.parquet")
+    expect = pd.read_parquet(fpath)
+    assert_eq(expect, got)
```

Comment:
Is there any way to generate this data via pyarrow or something similar as part of the tests instead of having to check in the test files?